README.md

```diff
@@ -24,39 +24,30 @@ The OBS source code repository is hosted on [Github](http://github.com/opensuse/
         src/api       Rails app (Ruby on Rails)
         src/backend   Backend code (Perl)
 
-## Setup
-There are 3 scenarios for which you can setup an OBS instance.
+## Installation
+To run the OBS in production we recommend using our [appliance](http://openbuildservice.org/download/) which is the whole package: A recent and stable Linux Operating System ([openSUSE](http://www.opensuse.org)) bundled and pre-configured with all the server and OBS components you need to get going.
 
-1. Use the OBS appliance
+If that is not for you because you have some special needs for your setup (e.g. different partition schema, SLES as base system, etc.) you can also install our packages and run a setup wizard.
 
-  To run the OBS in production we recommend using our [OBS appliance](http://openbuildservice.org/download/) which is the whole package:
-  a recent and stable Linux Operating System ([openSUSE](http://www.opensuse.org)) bundled and pre-configured with all the server and
-  OBS components you need to get going.
+After finishing the installation of your base system, follow these steps:
 
-2. Install packages and run the setup wizard
+1. Add the OBS software repository with zypper. Please be aware, that the needed URL differs, depending on your Base Operating System. We use openSUSE Leap 42.1 in this example.
 
-  This installation method is recommended if you have some special needs for your setup (e.g. different partition schema, SLES as base system, etc.).
+    ```shell
+    zypper ar -f http://download.opensuse.org/repositories/OBS:/Server:/2.7/openSUSE_42.1/OBS:Server:2.7.repo
+    ```
 
-  After finishing the installation of your base system, follow the following steps:
+2. Install the package
 
-  1. Add the OBS software repository with zypper
+   ```shell
+   zypper in obs-api obs-server
```

Comment:
We should either add the optional packages here or point to the pattern.

Additional packages are:   obs-source_service obs-signd

Alternative solution would be to use the pattern:

 zypper in -t pattern OBS_Server

Maybe the better solution, so the documentation can stay unchanged on package changes.